masterlist.yaml

```diff
@@ -247,6 +247,46 @@ common:
     content:
       - lang: en
         text: 'You seem to be using %1%, but you have not enabled a compatibility patch for this mod. A third party patch is available here: %2%'
+  - &patchRPC
```

Comment:
You could shorten those further by making a requiem version of the patch3rdParty text.

```yaml
  - &patchRequiem3rdParty
    type: warn
    content:
      - lang: en
        text: 'You seem to be using Requiem, but you have not enabled a compatibility patch for this mod. A third party patch is available here: %1%'
```

I think the names could include `Requiem` just for clarity.

```yaml
  - &patchRequiemRPC
    <<: *patch3rdParty
    subs: [ '[Requiem Patch Central](https://www.nexusmods.com/skyrim/mods/61621)' ]

  - &patchRequiemNRM
    <<: *patch3rdParty
    subs: [ '[NRM Requiem Patches](https://www.nexusmods.com/skyrim/mods/60508)' ]

  - &patchRequiemMRP
    <<: *patch3rdParty
    subs: [ '[NRM Requiem Patches](https://www.nexusmods.com/skyrim/mods/60508)' ]

  - &patchRequiemKFR
    <<: *patch3rdParty
    subs: [ '[KFR - Kryptopyr''s Fixes Reqtified](https://www.nexusmods.com/skyrim/mods/61520)' ]

```

Reply:
> Any changes to patch3rdParty including translations must then be applied twice.

Well you'd still be using the same message except instead of a sub your adding it directly.
The translation would still be the same, regardless of a sub being used or not.

> If every mod that has multiple third-party patches has its own version of this message, this can get quite inconvenient.

My reasoning would be that instead of each page having a `common` entry.
You would use `patchRequiem3rdParty` for pages with a few patches.
`Requiem Patch Central` is good to have as a common entry since it has 40+ entries.

So instead of each page having a common entry.
```yaml
  - &patchRequiemNRM
    <<: *patch3rdParty
    subs:
      - 'Requiem'
      - '[NRM Requiem Patches](https://www.nexusmods.com/skyrim/mods/60508)'
```

You would use for `patchRequiem3rdParty` pages with only a few patches.
```yaml
  - &patchRequiem3rdParty
    type: warn
    content:
      - lang: en
        text: 'You seem to be using Requiem, but you have not enabled a compatibility patch for this mod. A third party patch is available here: %1%'
```
```yaml
  - <<: *patchRequiem3rdParty
    subs: [ '[NRM Requiem Patches](https://www.nexusmods.com/skyrim/mods/60508)' ]
```

Reply:
I see your point.

All requested changes are now pushed.